src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,49 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
```

Comment:
This method is too complicated and it is difficult to read and understand. I suggest you to break it down into several methods. for example: readFile, calculateStatistics, generateReport and writeToFile. After that code will be much more readable, reusable and maintainable